src/config.js

```diff
@@ -8,6 +8,11 @@ export class Config {
   configurations = {}
 
   constructor(viewManager, viewManagerConfig) {
+    this.configure({
+      labelFormat: element => {
```

Comment:
i.d do 
`this.configurations.labelFormat = element => ....`
but again, matter of taste (though it'll be 'faster')